### PR TITLE
first version of conanfile.py, issue#38

### LINKED ISSUE #38: Provide conan packet for simple installation
The documentation is available here: https://docs.conan.io/en/latest/creating_packages.html

### PR DESCRIPTION
**Pull request by dpronin**
Provide first version of conanfile.py

Resolves [issue#38](https://github.com/erikzenker/inotify-cpp/issues/38)

Signed-off-by: Denis Pronin <dannftk@yandex.ru>

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from conans import ConanFile, CMake, tools
+
+class INotifyCpp(ConanFile):
+    name = "inotifycpp"
```
Comment (erikzenker) [inline conanfile.py:4]: it is `inotify-cpp`

**Review by erikzenker — CHANGES_REQUESTED** [review]
Thank you for your contribution :-)

File: test/unit/CMakeLists.txt [hunk 2]
```diff
@@ -30,7 +21,7 @@ target_compile_features(inotify_unit_test PRIVATE cxx_std_17)
 target_link_libraries(inotify_unit_test
         PRIVATE
           inotify-cpp::inotify-cpp
-          Boost::unit_test_framework
+          CONAN_PKG::boost
```
[the remark this answers is not in the document]
Comment (dannftk) [reply test/unit/CMakeLists.txt:24]: Where have you seen such an approach?

[the remark this answers is not in the document]
Comment (erikzenker) [reply test/unit/CMakeLists.txt:24]: https://docs.conan.io/en/latest/integrations/build_system/cmake/cmake_find_package_generator.html

**Pull request closed without merging**